Options.py

```diff
@@ -835,14 +835,21 @@ def __contains__(self, item):
 
 class Accessibility(Choice):
     """Set rules for reachability of your items/locations.
-    Locations: ensure everything can be reached and acquired.
-    Items: ensure all logically relevant items can be acquired.
+    Full: ensure everything can be reached and acquired.
     Minimal: ensure what is needed to reach your goal can be acquired."""
     display_name = "Accessibility"
-    option_locations = 0
-    option_items = 1
+    option_full = 0
     option_minimal = 2
     alias_none = 2
+    default = 0
+
+
+class ItemsAccessibility(Accessibility):
+    """Set rules for reachability of your items/locations.
+    Full: ensure everything can be reached and acquired.
+    Items: ensure all logically relevant items can be acquired. Some items, such as keys, may be self-locking.
```

Comment:
I was hoping this option could allow, in some games, some locations which are not accessible, whether they are "self-locking" or not. SM used to force `items` accessibility if it had unreachable locations due to fun settings removing advancement items, and I'd like to see it return to that, and I'd like to do something similar in an upcoming update.

Reply:
I didn't change the behavior at all. This is how games that use items accessibility have always used it. You can of course override this class and its description though.

Reply:
I wonder whether that should be a different subclass of `Accessibility` - one that says "ensure all progression items in the game can be acquired".

This would make sure that only trash items can be in a location that can't be reached.